Source/bmalloc/libpas/src/libpas/pas_utils.c

```diff
@@ -147,7 +147,18 @@ PAS_NEVER_INLINE PAS_NO_RETURN static void pas_crash_with_info_impl(uint64_t rea
 
 #else
 
-PAS_NEVER_INLINE PAS_NO_RETURN static void pas_crash_with_info_impl(uint64_t, uint64_t, uint64_t, uint64_t, uint64_t, uint64_t, uint64_t) { __builtin_trap(); }
+PAS_NEVER_INLINE PAS_NO_RETURN static void pas_crash_with_info_impl(uint64_t reason, uint64_t misc1, uint64_t misc2, uint64_t misc3, uint64_t misc4, uint64_t misc5, uint64_t misc6)
```

Comment:
Let's suppress warnings instead. We strongly want to keep this function having only `__builtin_trap`.

Reply:
OK, let's use `PAS_IGNORE_GCC_WARNINGS_{BEGIN,END}`